src/auth/.gcb/builds/triggers/main.tf

```diff
@@ -236,3 +244,43 @@ resource "google_cloudbuild_trigger" "post-merge" {
 
   include_build_logs = "INCLUDE_BUILD_LOGS_WITH_STATUS"
 }
+
+resource "google_pubsub_topic" "terraform_runner_topic" {
+  name = "terraform-runner"
+}
+
+resource "google_pubsub_subscription" "terraform_runner_sub" {
+  name  = "terraform-sub"
+  topic = google_pubsub_topic.terraform_runner_topic.name
+}
+
+resource "google_cloud_scheduler_job" "job" {
+  name        = "terraform-job"
+  description = "Periodically sync terraform build"
+  schedule    = "0 0 1 * *" # Midnight on 1st day of the month	
+
+  pubsub_target {
+    topic_name = google_pubsub_topic.terraform_runner_topic.id
+    data       = base64encode("sync")
+  }
+}
+
+resource "google_cloudbuild_trigger" "pubsub-trigger" {
+  for_each = tomap(local.pubsub_builds)
```

Comment:
No need for this to be a loop.

We use a loop for the integration build so we can have a triggers that runs on PR and one that runs on merge.